### PR TITLE
exercises + tour of go exercises

### PR DESCRIPTION
**Pull request by temaniarpit27**

### CODE REVIEW

File: course-exercises/slices/1.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package slices
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/temaniarpit27/polygon-core-developer-course/helper"
+)
+
+func Exercise1(arr []int, n int) []int {
```
Comment (JekaMas) [inline course-exercises/slices/1.go]: Could you give function names according to their's functionality and just left comments about numbers of exercises?

File: course-exercises/maps/1.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import (
+	"strings"
+
+	"github.com/temaniarpit27/polygon-core-developer-course/helper"
+)
+
+func WordCount(s string) map[string]int {
+	if s == "" {
+		return make(map[string]int, 0)
```
Comment (JekaMas) [inline course-exercises/maps/1.go:11]: .

File: course-exercises/slices/1.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+func Exercise2(arr []int, n int) []int {
+	arr = append(arr, n)
+	return arr
+}
+
+func Exercise3(arr []int, n int) []int {
+	resp := make([]int, 0)
+	resp = append(resp, n)
+	resp = append(resp, arr...)
```
Comment (JekaMas) [inline course-exercises/slices/1.go]: You also could use copy function.

File: course-exercises/maps/1.go (hunk 2 above)
Comment (temaniarpit27) [reply course-exercises/maps/1.go:11]: didn't get this

File: course-exercises/slices/1.go (hunk 3 above)
Comment (temaniarpit27) [reply course-exercises/slices/1.go]: check now

File: course-exercises/slices/1.go (hunk 1 above)
Comment (temaniarpit27) [reply course-exercises/slices/1.go]: done

**Review by marcello33 — APPROVED** [review]

**Pull request merged**